### DISCUSSION TITLE
Being able to host worlds and have a friends list like old console edition

### DISCUSSION THREAD

**Discussion by Big1N** [Nostalgica-Reverie/Content-Monorepo#115] [Ideas]
I have a good suggestion and you Should totally add a mod called essential or make a mod similar so we can host worlds and have a friends list. 
![IMG_0275](https://github.com/user-attachments/assets/c993c330-b0ce-4b0b-96b0-7d55f755c625)

**Comment by DBTDerpbox** [comment]
World host is already in reconsole? Legacy4J directly supports it

**Comment by omo50** [comment]
as Emmie said
